### PR TITLE
Updated README.md file

### PR DESCRIPTION
**Pull request by Sophyia7**
I updated the README.md file by fixing spelling errors and adding missing images.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -373,8 +373,8 @@ let seed: SeededAsset = {
 ```
 
 
-### `/assets/create`
```
Comment (jordan-pieces) [inline README.md:376]: this one is okay to stay like this: `/assets/create`

Comment (jordan-pieces) [reply README.md:376]: correct: /assets/create
incorrect: Assets/create

File: README.md [hunk 2]
```diff
@@ -304,13 +304,13 @@ npm run start
 
 And you should have the same content in the main browser window as before once this completes. But if you open up your inspector using `cmd+option+i` or `ctrl+shift+c` you will see this inside of your console:
 
-![[Screenshot 2023-10-24 at 4.45.49 PM.png]]
+![Console Output](image.png)
```
Comment (jordan-pieces) [inline README.md:307]: i created an assets folder that i will be merging in, but you can just create one as well and put it at the root directory of the project

File: README.md [hunk 3]
```diff
@@ -304,13 +304,13 @@ npm run start
...
 
 This includes both the full OS response object with all the data that you will need to get going with other endpoints, and your application is now connected and ready to go for the rest of your exploration and discovery!
 
 Follow along with these next steps to learn about **assets and formats,** two things that are very important for managing any form of data with Pieces OS.
 
-## Getting Started with `/asset` + `/assets`
```
Comment (jordan-pieces) [inline README.md:313]: Correct: Asset or /assets
Incorrect: Assets or /asset

File: README.md [hunk 4]
```diff
@@ -912,7 +912,7 @@ Now when you follow this guide, you will be receiving this data back from inside
 This is a very simple guide on how to get up and running using the @pieces.app/client npm package and create a web environment that you can build on top of. **Fork this repo** to get started and learn about the depth of possibilities you have with Pieces OS.
 
 More guides will be coming soon around:
-- Adding a UI to view multiple snippets from `/assets`
```
Comment (jordan-pieces) [inline README.md:915]: this is okay as /assets

**Review by jordan-pieces — CHANGES_REQUESTED** [review]
my mistake on the /assets vs Asset difference, i could've been more clear about the importance in their differences. Asset is a type, while Assets[<Asset>] is an array of your assets.

File: README.md (hunk 4 above)
Comment (Sophyia7) [reply README.md:915]: @jordan-pieces I have made the changes.

**Review by jordan-pieces — APPROVED** [review]
looking good thank you @Sophyia7 !

**Pull request merged**